pkg/skaffold/build/gcb/types.go

```diff
@@ -58,8 +59,26 @@ const (
 
 	// RetryDelay is the time to wait in between polling the status of the cloud build
 	RetryDelay = 1 * time.Second
+
+	// BackoffFactor is the exponent for exponential backoff during build status polling
+	BackoffFactor = 1.5
+
+	// BackoffSteps is the number of times we increase the backoff time during exponential backoff
+	BackoffSteps = 10
+
+	// RetryTimeout is the max amount of time to retry getting the status of the build before erroring
+	RetryTimeout = 3 * time.Minute
 )
 
+func NewStatusBackoff() *wait.Backoff {
+	return &wait.Backoff{
```

Comment:
nice find!

Reply:
can't believe there's nothing like this in the stdlib... :man_shrugging: